app/views/static_pages/home.html.erb

```diff
@@ -1,10 +1,27 @@
 <% provide :title, t(".title") %>
-<div class="center jumbotron">
-  <h1><%= t ".title" %></h1>
-  <h2>
-    <%= t ".description" %>
-    <%= t(".links.rt_home_page_html", href: link_to(t(".rt_home_page_text"), Settings.links.rails_tutorial.home_page)) %>
-  </h2>
-  <%= link_to t(".sign_up"), signup_path, class: "btn btn-lg btn-primary" %>
-</div>
-<%= link_to image_tag("rails.svg", alt: "Rails logo", width: "#{Settings.size.logo}px"), "https://rubyonrails.org/" %>
+<% if logged_in? %>
+  <div class="row">
+    <aside class="col-md-4">
+      <section class="user_info">
+        <%= render "shared/user_info" %>
+      </section>
+      <section class="micropost_form">
+        <%= render "shared/micropost_form" %>
+      </section>
+    </aside>
+    <div class="col-md-8">
+      <h3><%= t(".feed") %></h3>
+      <%= render "shared/feed" %>
+    </div>
+  </div>
+<% else %>
```

Comment:
nếu người dùng đăng nhập thì sẽ in ra thông tin người dùng, bài viết, và form để tạo bài viết